ui/apps/platform/src/routePaths.js

```diff
@@ -149,7 +149,7 @@ const vulnManagementPathToLabelMap = {
 export const basePathToLabelMap = {
     [dashboardPath]: 'Dashboard',
     [networkBasePath]: 'Network Graph',
-    [networkBasePathPF]: 'Network Graph-Patternfly',
+    [networkBasePathPF]: 'PatternFly Network Graph',
```

Comment:
Because the container renders the page title, which is the preferred pattern:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/NetworkGraph/NetworkGraphPage.tsx#L65

Can you see if it works to delete this title property and instead provide a literal string in the `NavigationSidebar` component? Although the procedure to add a container must include `Body` and usually includes `NavigationSidebar` components, every extra “global registration” step is a barrier to inclusion and risks being the straw that breaks the camel’s brain, according to the principle of up-to-7 items in short-term memory.